### PR TITLE
Crypto routes and chatbot small screen bugfix

### PR DESCRIPTION
**Pull request by Krysh90**

### CODE REVIEW

File: app/screens/home/HomeScreen.tsx [hunk 1]
```diff
@@ -174,6 +197,7 @@ const HomeScreen = ({ session, settings }: { session?: Session; settings?: AppSe
     setUser(undefined);
     setBuyRoutes(undefined);
     setSellRoutes(undefined);
+    // Krysh: should we reset staking & crypto routes too?
```
Comment (davidleomay) [inline app/screens/home/HomeScreen.tsx:200]: Missing

File: app/screens/home/RouteList.tsx [hunk 2]
```diff
@@ -440,15 +500,27 @@ const RouteList = ({
...
+                {t("model.route.crypto")}
+              </DeFiButton>
           </View>
+          ) : (
+            <View style={AppStyles.containerHorizontal}>
+              <DeFiButton mode="contained" onPress={() => setIsStakingRouteEdit(true)} style={{ flex: 1 }}>
+                {t("model.route.staking")}
+              </DeFiButton>
+            </View>
+          )}
```
Comment (davidleomay) [inline app/screens/home/RouteList.tsx:519]: Is it required to duplicate the staking button? Or could you just put the `isBetaUser` around the Crypto button?

Comment (Krysh90) [reply app/screens/home/RouteList.tsx:519]: good point. this is unnecessary

File: app/screens/home/HomeScreen.tsx (hunk 1 above)
Comment (Krysh90) [reply app/screens/home/HomeScreen.tsx:200]: true that

Comment (Krysh90) [reply app/screens/home/HomeScreen.tsx:200]: resolve in #125

File: app/screens/home/RouteList.tsx (hunk 2 above)
Comment (Krysh90) [reply app/screens/home/RouteList.tsx:519]: resolve in #125

**Review by davidleomay — APPROVED** [review]

**Pull request merged**